adapters/adapterstest/test_json.go

```diff
@@ -318,6 +320,14 @@ func diffHttpRequests(description string, actual *adapters.RequestData, expected
 			return err
 		}
 	}
+
+	if len(expected.ImpIDs) < 1 {
+		return fmt.Errorf(`expected.ImpIDs must contain at least one imp ID`)
+	}
+
+	if !reflect.DeepEqual(expected.ImpIDs, actual.ImpIDs) {
+		return fmt.Errorf(`%s actual.ImpIDs "%q" do not match expected "%q"`, description, actual.ImpIDs, expected.ImpIDs)
+	}
```

Comment:
Actually could you please change this block to the following:
```
opt := cmpopts.SortSlices(func(a, b string) bool { return a < b })
if !cmp.Equal(expected.ImpIDs, actual.ImpIDs, opt) {
	return fmt.Errorf(`%s actual.ImpIDs "%q" do not match expected "%q"`, description, actual.ImpIDs, expected.ImpIDs)
}
```

My original suggestion here considers order when determining equality and we should ignore that.

And here are the imports needed to make this work:
```
"github.com/google/go-cmp/cmp"
"github.com/google/go-cmp/cmp/cmpopts"
```